scene/gui/graph_node.cpp

```diff
@@ -366,38 +366,40 @@ void GraphNode::_notification(int p_what) {
 				close_rect = Rect2();
 			}
 
-			for (const KeyValue<int, Slot> &E : slot_info) {
-				if (E.key < 0 || E.key >= cache_y.size()) {
-					continue;
-				}
-				if (!slot_info.has(E.key)) {
-					continue;
-				}
-				const Slot &s = slot_info[E.key];
-				// Left port.
-				if (s.enable_left) {
-					Ref<Texture2D> p = port;
-					if (s.custom_slot_left.is_valid()) {
-						p = s.custom_slot_left;
+			if(get_child_count() > 0){
```

Comment:
Style issue. Please make sure to use the same style as surrounding code (or better, use clang-format locally to enforce it). See https://docs.godotengine.org/en/latest/community/contributing/code_style_guidelines.html
```suggestion
			if (get_child_count() > 0) {
```

Reply:
@YuriSizov @akien-mga, I have update the PR and modified the style issue with clang-format. Thanks for your help again, i really learn a lot during the pr processing.